src/platforms/platform.dom.js

```diff
@@ -277,13 +277,10 @@ function addResizeListener(node, listener, chart) {
 			var container = aspectRatio && node.parentNode;
 			var w = container ? container.clientWidth : 0;
 			var ret = listener(createEvent('resize', chart));
-			if (container) {
-				expando._width = container.clientWidth;
-				if (expando._width !== w && chart.canvas) {
-					// If the container size changed during chart resize, we can assume scrollbar appeared.
-					// So let's resize again, with the scrollbar visible
-					ret = listener(createEvent('resize', chart));
-				}
+			if (container && container.clientWidth !== w && chart.canvas) {
+				// If the container size changed during chart resize, we can assume scrollbar appeared.
+				// So let's resize again, with the scrollbar visible
+				ret = listener(createEvent('resize', chart));
```

Comment:
I'm curious how the 2nd resize doesn't cause the scrollbar to go away

Reply:
What _I think_ happens: We are doing the resize in a `onScroll` event. We resize, notice it made scrollbars visible and resize again. Scrollbars are hidden in the end of this scroll event. And I suppose browsers won't fire another scroll until the previous is done.

Initially I was string the width, but accidentaly found out, it was not really needed.

Reply:
Ahh, that's a good find. If `throttled()` can prevent the double event then I'm ok with testing this out further